### PR TITLE
Participant website

### PR DESCRIPTION
**Pull request by AlexanderFengFeng**
Majority of changes from google doc provided by Dom

### CODE REVIEW

File: amgut/handlers/introduction.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from amgut.handlers.base_handlers import BaseHandler
+
+
+class IntroductionHandler(BaseHandler):
+    page = 'introduction.html'
+
+    def get(self):
+        self.render(self.page, loginerror='')
```
Comment (AlexanderFengFeng) [inline amgut/handlers/introduction.py]: Created introduction page to to transfer American Gut 101 information from FAQ

File: amgut/lib/locale_data/british_gut.py [hunk 2]
```diff
@@ -1037,12 +1066,14 @@
...
 # Actual text locale
 text_locale = {
     'nojs.html': _NOJS,
     '404.html': _404,
     '403.html': _403,
     'FAQ.html': _FAQ,
+    'introduction.html': _INTRODUCTION,
     'new_participant_overview.html': _NEW_PARTICIPANT_OVERVIEW,
     'personal_microbiome_overview.html': _PERSONAL_MICROBIOME,
     'addendum.html': _ADDENDUM,
```
Comment (AlexanderFengFeng) [inline amgut/lib/locale_data/british_gut.py]: All the changes SHOULD be the same as the AG text locale.

File: amgut/templates/addendum.html [hunk 3]
```diff
@@ -117,9 +117,10 @@
...
     <div class="faq">
         <div id = 'text' align="justify">
-            <a name = "top"><h1>{% raw tl['TITLE'] %}</h1></a>
+            <a name = "top"><h1 style='text-align:center;'>{% raw tl['TITLE'] %}</h1></a>
             <p>{% raw tl['INTRO'] %}</p>
             <h3>{% raw tl['LEARN_MORE'] %}</h3>
+            <p style="font-style: italic;">{% raw tl['NOTE'] %}</p>
             <table class = 'certif'>
                 <tr>            
                     <td colspan = '9'>
```
Comment (AlexanderFengFeng) [inline amgut/templates/addendum.html]: Centered addendum title

File: amgut/templates/addendum.html [hunk 4]
```diff
@@ -117,9 +117,10 @@
 <div class="faq_wrapper">
     <div class="faq">
         <div id = 'text' align="justify">
-            <a name = "top"><h1>{% raw tl['TITLE'] %}</h1></a>
+            <a name = "top"><h1 style='text-align:center;'>{% raw tl['TITLE'] %}</h1></a>
             <p>{% raw tl['INTRO'] %}</p>
             <h3>{% raw tl['LEARN_MORE'] %}</h3>
+            <p style="font-style: italic;">{% raw tl['NOTE'] %}</p>
```
Comment (AlexanderFengFeng) [inline amgut/templates/addendum.html]: Added clarifying sentence that the results are strictly an example

File: amgut/templates/no_auth_sitebase.html [hunk 5]
```diff
@@ -103,19 +104,30 @@
...
+                        <ul>
+                            <li><a href="{% raw media_locale['AMERICAN_GUT_PROJECT'] %}">{% raw media_locale['NAV_AMERICAN_GUT_PROJECT'] %}</a></li>
+                            <li><a href="{% raw media_locale['BRITISH_GUT_PROJECT'] %}">{% raw media_locale['NAV_BRITISH_GUT_PROJECT'] %}</a></li>
+                        {% for alt_site in media_locale['GUT_PROJECT_SHORTHANDS'] %}
+                            {% if alt_site != media_locale['SHORTHAND'] %}
+                            <li><a href="{% raw media_locale['ALT_GUT_SITE'] %}/{% raw alt_site.replace(' ','') %}">{% raw alt_site %}{% raw media_locale['NAV_ALT_GUT'] %}</a></li>
+                            {% end %}
+                        {% end %}
+                        </ul>
+                    </li>
```
Comment (AlexanderFengFeng) [inline amgut/templates/no_auth_sitebase.html]: Added "Other sites" tab

File: amgut/templates/portal.html [hunk 6]
```diff
@@ -110,10 +122,11 @@ <h2 style="color:red">{% raw tl['UNCONSENTED_HEADER'] %}</h2>
               </ul>
             {% end %}
             <h2>{% raw tl['SAMPLE_STEPS_HEADER_1'] %}</h2>
-            <p class="portal_blurb">{% raw tl['SAMPLE_STEPS_TEXT_1'] %}</p>
-            <ol class="sample_steps">{% raw tl['SAMPLE_STEPS_TEXT_2'] %}</ol>
-            <p class="portal_blurb">{% raw tl['SAMPLE_STEPS_TEXT_3'] % fmt %}</p>
+            <p class="portal_blurb">{% raw tl['SAMPLE_STEPS_TEXT_1'] % fmt %}</p>
+            <p class="portal_blurb">{% raw tl['SAMPLE_STEPS_TEXT_2'] %}</p>
+            <ol class="sample_steps">{% raw tl['SAMPLE_STEPS_TEXT_3'] %}</ol>
             <h2>{% raw tl['SAMPLE_STEPS_HEADER_2'] %}</h2>
```
Comment (AlexanderFengFeng) [inline amgut/templates/portal.html]: Rearranged order of sentences in portal based on suggestion

File: amgut/templates/portal.html [hunk 7]
```diff
@@ -110,10 +122,11 @@ <h2 style="color:red">{% raw tl['UNCONSENTED_HEADER'] %}</h2>
...
+            <p class ="portal_blurb">{% raw tl['FOBT_INSTRUCTIONS'] %}</p>
             <p class="portal_blurb">{% raw tl['SAMPLE_STEPS_TEXT_4'] %}</p>
```
Comment (AlexanderFengFeng) [inline amgut/templates/portal.html]: Added FOBT instructions to portal

File: amgut/templates/portal.html [hunk 8]
```diff
@@ -143,6 +156,13 @@ <h2>{% raw tl['LOG_SAMPLE_HEADER_1'] %}</h2>
...
             <div class="centered"><img src="{% raw media_locale['SITEBASE'] %}/static/img/labeled_tube.png"></div>
             <p class="portal_blurb">{% raw tl['LOG_SAMPLE_TEXT_2'] %}</p>
+                <h2><font color="red">{% raw tl['LOG_SAMPLE_NEW'] %}</font></h2>
+                <ul>
+                {% for barcode in barcodes %}
+                    <li><a href="{% raw media_locale['SITEBASE'] %}/authed/add_sample_overview/">{{barcode}}</a></li>
+                {% end %}
+                </ul>
+            <p class="portal_blurb">{% raw tl['LOG_SAMPLE_TEXT_3'] %}</p>
         </div>
```
Comment (AlexanderFengFeng) [inline amgut/templates/portal.html]: Now displays barcodes that need to be assigned. Clicking on a barcode will redirect to the add sample overview site

**Comment by wasade** [comment]
This is great! only comment, for the logos, would it be possible to use the TM versions?

https://www.dropbox.com/s/sv6wsqo7mt3famz/ag.jpg?dl=0
https://www.dropbox.com/s/qvowqmm862cyhjb/bg.jpg?dl=0

**Comment by AlexanderFengFeng** [comment]
@wasade These right? I think I added it as per Dom's request.
screen shot 2018-07-26 at 11 03 42 pm

**Comment by AlexanderFengFeng** [comment]
@wasade Got the conflicts resolved so I hope it'd be good to go after you resolve the issue with the dev base.

**Comment by wasade** [comment]
Great thanks!

**Pull request merged**